koku/api/report/ocp/provider_map.py

```diff
@@ -977,6 +1056,22 @@ def __init__(self, provider, report_type, schema_name):
             "virtual_machines": {
                 "default": OCPVirtualMachineSummaryP,
             },
+            "gpu": {
+                "default": OCPGpuSummaryP,
+                ("cluster",): OCPGpuSummaryP,
```

Comment:
If they are all the same table, you can just set a single default like what we do with virtual_machines up above.